### DISCUSSION TITLE
[Non technical issue] Question

### DISCUSSION THREAD

**Discussion by ghost** [Refloow/Steam-Card-Bot-PRO#45] [General]
I can't understand what it's means,my english is not very good.
![屏幕截图 2022-02-11 030522](https://user-images.githubusercontent.com/99354599/153479050-827bde79-f895-489b-bbba-ba8d9a9a6f13.png)

**Comment by Refloow** [comment]
> I can't understand what it's means,my english is not very good. ![屏幕截图 2022-02-11 030522](https://user-images.githubusercontent.com/99354599/153479050-827bde79-f895-489b-bbba-ba8d9a9a6f13.png)

Please do not open issues for non technical reports.
I can't help you unless you know english.

That setting there sets status message as game that is playin on the account.
